### ISSUE TITLE
GCS: Notify when a reboot is pending (due to settings changes)

### ISSUE THREAD

**Issue by tracernz** [d-ronin/dronin#76]
Add a notification and reboot button at the bottom of the config widget to advise when a reboot is pending due to config changes. This should be a big win for a common newbie frustration.

**Comment by jpbarraca** [comment]
@tracernz It would be very useful. This involves other widgets besides Board and Modules?

**Comment by tracernz** [comment]
Off hand just the hardware settings and modules UAVOs, there are probably others too.

**Comment by mlyle** [comment]
We could add a property on UAVO fields "requiresReboot", that when changed and stored a reboot is flagged required.  hardware settings, modules, and RGB LED count are definitely some.

**Comment by tracernz** [comment]
Yes, that could work. Maybe a messagebox when you hit save asking if you want to reboot.. or just set an icon on the reboot button if we want less intrusive/annoying. I think I prefer the latter.

**Comment by mlyle** [comment]
I was kinda thinking something down next to the telemetry widget, so it "follows you".

**Comment by mlyle** [comment]
How about instead of uavo fields in UI, something that the uavogen understands?  Easy to add a tag <applyingRequiresReset/>

**Comment by tracernz** [comment]
Yes, that is the right place to add "requires_reboot" information.
